lib/tackle/consumer.ex

```diff
@@ -13,6 +13,8 @@ defmodule Tackle.Consumer do
     retry_delay = options[:retry_delay] || 10
     retry_limit = options[:retry_limit] || 10
 
+    prefetch_count = options[:prefetch_count]
```

Comment:
Because `Tackle.Channel` is used in number of places.
So, if not set in it, default would have to be set in more than one place.
Which I wanted to avoid.

I can add default in consumer also?